mlos_bench/mlos_bench/environments/remote/remote_env.py

```diff
@@ -72,6 +75,7 @@ def __init__(  # pylint: disable=too-many-arguments
         )
 
         self._wait_boot = self.config.get("wait_boot", False)
+        self._command_prefix = self._RE_SPECIAL.sub("-", self.name).lower() + "-"
```

Comment:
```suggestion
        self._command_prefix = "mlos-" + self._RE_SPECIAL.sub("-", self.name).lower() + "-"
```

Just to help namespace them a tad more.